tsconfig.json

```diff
@@ -15,7 +15,7 @@
     "paths": {
       "~/*": ["./app/*"]
     },
-
+    "typeRoots": ["./node_modules/@types"],
```

Comment:
What is this one for?

Reply:
I was getting an odd error 'Cannot find type definition file for 'istanbul-reports'.' and this really was a last ditch attempt to get that error resolved.

Reply:
Odd I'm not seeing that with the same dependencies. Have you tried deleting your node_modules folder and running npm install again?